src/Comonicon.jl

```diff
@@ -9,6 +9,7 @@ module Comonicon
 using Pkg
 using Markdown
 using ExproniconLite
+import OrderedCollections: OrderedDict
```

Comment:
```suggestion
using OrderedCollections: OrderedDict
```

`import` should not have been provided in Julia :P, it's terrible, let's try to use `using` only.